modules/programs/rofi.nix

```diff
@@ -72,20 +72,14 @@ let
   };
 
   valueToString = value:
-    if isBool value then (if value then "true" else "else") else toString value;
```

Comment:
This was a bug in the old code, when you set the value to `false` it wrote `else` to config instead.